WARNNING: Disclaimer: DFS specification is not supported in China And Other DFS Restricted Countries

diff --git a/src/ap/dfs.c b/src/ap/dfs.c
index b990fb3..ed39d97 100644
--- a/src/ap/dfs.c
+++ b/src/ap/dfs.c
@@ -826,6 +826,8 @@ int hostapd_handle_dfs(struct hostapd_iface *iface)
 		}
 	} while (res);

+    iface->dfs_cac_ms=1000;
+
 	/* Finally start CAC */
 	hostapd_set_state(iface, HAPD_IFACE_DFS);
 	wpa_printf(MSG_DEBUG, "DFS start CAC on %d MHz", iface->freq);
